shared/chat/conversation/list-area/normal/index.desktop.tsx

```diff
@@ -624,11 +624,19 @@ class OrdinalWaypoint extends React.Component<OrdinalWaypointProps, OrdinalWaypo
     return null
   }
 
+  // Cache rendered children if the ordinals are the same, else we'll thrash a lot as we scroll up and down
+  private lastVisibleChildrenOrdinals: Array<Types.Ordinal> = []
+  private lastVisibleChildren: React.ReactNode = null
```

Comment:
Shouldn't this be a SCU instead? This seems hacky

Reply:
no cause it does change. we just don't need to call render again